.github/workflows/service_test_sled.yml

```diff
@@ -53,3 +53,14 @@ jobs:
           OPENDAL_SLED_TEST: on
           OPENDAL_SLED_ROOT: /
           OPENDAL_SLED_DATADIR: /tmp/opendal/sled/
+      - name: Test sled with tree
```

Comment:
Plead add a new job called `sled-with-tree` so that they can be test concurrently.